public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -59,57 +59,43 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
     <Page navModel={navModel} pageNav={pageNav} layout={PageLayoutType.Custom}>
       {editPanel && <editPanel.Component model={editPanel} />}
       {!editPanel && (
-        <div className={cx(styles.pageContainer, hasControls && styles.pageContainerWithControls)}>
-          <NavToolbarActions dashboard={model} />
-          {controls && (
-            <div className={styles.controlsWrapper}>
-              <controls.Component model={controls} />
-            </div>
-          )}
-          <PanelsContainer
-            // This id is used by the image renderer to scroll through the dashboard
-            id="page-scrollbar"
-            className={styles.panelsContainer}
-            testId={selectors.pages.Dashboard.DashNav.scrollContainer}
```

Comment:
the e2e test that's currently failing relies on this `testId`

i think that e2e test is faulty anyway. it's attempting to test restoring the scroll position, but we know that doesn't work in scenes so it should never be passing

the problem is that e2e test uses `.toBeVisible()`, which in cypress does not necessarily mean it's within the viewport